modules/DMZ/data_kit/data_splitting.py

```diff
@@ -29,14 +34,14 @@ def get_train_test(pandas_data, target_col):
     return cross_validation.train_test_split(x, y, test_size=0.2)
 
 
-# Removes the target column from the input data.
+# Removes the target column from the input data. Returns DataFrames
 def separate_target(pandas_data, target_col):
     # Selection of training/target data for validation and training.
-    data = pd.DataFrame.as_matrix(pandas_data)
-    target_loc = pandas_data.columns.get_loc(target_col)
-    y = data[:, target_loc]
-
     x = pandas_data.drop(target_col, 1)
-    x = pd.DataFrame.as_matrix(x)
+    y = pandas_data[[target_col]]
 
     return x, y
```

Comment:
x,y are poor variable names for this. Maybe something like predictors and response, or predictors and targets